lib/david/registry.rb

```diff
@@ -5,7 +5,7 @@ module Registry
     protected
 
     def log
-      @log ||= server.log
+      @log ||= Celluloid.logger
```

Comment:
Since we have two servers now and simply used the Celluoid logger here. Not sure if thats's a good idea though…